packages/gestalt/src/PageHeaderSubcomponents.js

```diff
@@ -0,0 +1,200 @@
+      accessibilityExpanded={helperIconButton.accessibilityExpanded}
+      accessibilityHaspopup
+      accessibilityLabel={helperIconButton.accessibilityLabel}
+      bgColor="lightGray"
+      icon="question-mark"
+      iconColor="darkGray"
+      onClick={helperIconButton.onClick}
+      size="xs"
+    />
+  ) : null;
+}
+
+export function PageHeaderSubtext({
+  subtext,
+  helperLink,
+}: {|
+  subtext?: string,
+  helperLink?: {|
+    text: string,
+    accessibilityLabel: string,
+    href: string,
+    onClick: ({|
+      event: SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement>,
+      dangerouslyDisableOnNavigation: () => void,
+    |}) => void,
+  |},
+|}): Node {
+  return (
+    <Box wrap display="none" smDisplay="block">
+      <Text inline>
+        {subtext}{' '}
+        {helperLink ? (
+          <Link
+            inline
+            accessibilityLabel={helperLink.accessibilityLabel}
+            href={helperLink.href}
+            onClick={helperLink.onClick}
+            target="blank"
+          >
+            <span style={{ textDecoration: 'underline' }}>{helperLink.text}</span>
+          </Link>
+        ) : null}
+      </Text>
+    </Box>
+  );
+}
+
+export function PageHeaderActionBlock({
+  primaryAction,
+  secondaryAction,
+  dropdownItems,
+  dropdownAccessibilityLabel = '',
+}: {|
+  primaryAction?: ActionType,
+  secondaryAction?: ActionType,
+  dropdownItems?: $ReadOnlyArray<Node>,
+  dropdownAccessibilityLabel?: string,
+|}): Node {
+  const [open, setOpen] = useState(false);
+  const anchorRef = useRef(null);
+
+  return (
+    <Fragment>
+      <Box display="none" mdDisplay="block">
+        <Flex gap={2}>
+          {/* 48px height needed to maintain proper sizing when action is a Link */}
+          <Box height={48} display="flex" alignItems="center">
+            {secondaryAction}
+          </Box>
+          <Box height={48} display="flex" alignItems="center">
+            {primaryAction}
+          </Box>
+        </Flex>
+      </Box>
+      {primaryAction ? (
+        <Box display="block" mdDisplay="none">
+          <IconButton
+            accessibilityControls="pageheader-dropdown"
+            accessibilityExpanded={open}
+            accessibilityHaspopup
+            accessibilityLabel={dropdownAccessibilityLabel}
+            icon="ellipsis"
+            iconColor="darkGray"
+            onClick={() => setOpen((prevVal) => !prevVal)}
+            ref={anchorRef}
+            selected={open}
+            size="lg"
+          />
+          {open && dropdownItems && (
+            <Dropdown
+              anchor={anchorRef.current}
+              id="pageheader-dropdown"
+              onDismiss={() => setOpen(false)}
+            >
+              {dropdownItems}
+            </Dropdown>
+          )}
+        </Box>
+      ) : null}
+    </Fragment>
+  );
+}
+
+export function PageHeaderItemsBlock({ items }: {| items?: $ReadOnlyArray<Node> |}): Node {
```

Comment:
This component should assume items will be defined, with a check at the callsite in PageHeader (probably checking to make sure items has length)

Reply:
DONE